README.md

```diff
@@ -1,8 +1,34 @@
-## IAVL+ Tree
+## 
 
-![Test](https://github.com/tendermint/iavl/workflows/Test/badge.svg?branch=master)
-![Lint](https://github.com/tendermint/iavl/workflows/Lint/badge.svg?branch=master)
-[![](https://godoc.org/github.com/tendermint/iavl?status.svg)](http://godoc.org/github.com/tendermint/iavl)
+<div align="center">
```

Comment:
I think I [agree with @tessr](https://github.com/tendermint/tm-db/pull/103#discussion_r434396546) here, I'd prefer keeping it as Markdown.